src/pages/apply.js

```diff
@@ -113,7 +114,16 @@ class ApplicationForm extends Component {
       this.setState({
         loading: false,
         applicationFormConfig: values[0],
```

Comment:
Might be worth using destructuring here to make this more readable in the future, i.e.
```
Promise.all([
      loadApplicationFormConfig,
      loadUserData,
      loadGeneralSettings,
      loadUserApplication,
    ]).then(([applicationFormConfig, userData, generalSettings, userApplication]) =>
        ....
    );
```
Granted, I think I originally wrote this so feel free to make an issue for this and assign it to me.

Reply:
https://github.com/BUUPE/interview/issues/132